cg/store/api/find_business_data.py

```diff
@@ -252,9 +252,16 @@ def find_family_by_name(self, name: str) -> Family:
         """Find a family by family name within a customer."""
         return self.Family.query.filter_by(name=name).first()
 
-    def find_samples(self, customer: Customer, name: str) -> Query:
-        """Find samples within a customer."""
-        return self._get_query(table=Sample).filter_by(customer=customer, name=name)
+    def get_samples_by_customer_and_name(self, customer: Customer, name: str) -> List[Sample]:
```

Comment:
I have not seen that many filter functions where we pass objects as arguments. Would it make more sense to only pass the actual property of the object that the filtering is done on? I think it is the customer_entry_id in this case.

Reply:
fixed